src/main/java/org/littletonrobotics/urcl/URCL.java

```diff
@@ -44,6 +47,11 @@ public class URCL {
   private static RawPublisher periodicPublisher;
   private static RawPublisher aliasesPublisher;
   private static Notifier notifier;
+  private static final DataLog datalog = DataLogManager.getLog();
+  private static RawLogEntry persistentLogEntry = new RawLogEntry(datalog, "URCL/Raw/Persistent", "",
```

Comment:
This is missing a leading slash, so AdvantageScope doesn't identify it as being in the same table as the other keys.